packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -848,7 +848,12 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
 
   AnimationController _cursorBlinkOpacityController;
 
-  final LayerLink _layerLink = LayerLink();
+  final LayerLink _toolbarLayerLink = LayerLink();
+
```

Comment:
nit: Just group these three together without blank lines in between since they are logically similar.